lua/wire/server/wirelib.lua

```diff
@@ -254,13 +254,6 @@ function WireLib.AdjustSpecialOutputs(ent, names, types, descs)
 
 	local ent_ports = ent.Outputs or {}
 
-	if ent_ports.wirelink then
```

Comment:
what is this change

Reply:
this reverts https://github.com/wiremod/wire/pull/2314. I'm pushing the gpulib removal undo without it. If you can reproduce this breaking anything we can continue the discussion here.